src/utils/thirdPartyConfig.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable sort-keys */
+
+import * as t from "io-ts";
+
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+
+import { fallback, jsonFromString } from "../types/commons";
+
+export type AuthenticationConfig = t.TypeOf<typeof AuthenticationConfig>;
+export const AuthenticationConfig = t.interface({
```

Comment:
i think we'll need also a field "key" to store the api key

Reply:
Fixed in [a87f0f6](https://github.com/pagopa/io-backend/pull/890/commits/a87f0f6b0a20f386e8d80c22afdadc2d5060cf3d)